src/faebryk/core/cpp/__init__.py

```diff
@@ -1,19 +1,24 @@
 # This file is part of the faebryk project
 # SPDX-License-Identifier: MIT
 
+import json
 import logging
 import pathlib
 import shutil
-import site
+from importlib.metadata import Distribution
 from typing import Callable
 
 logger = logging.getLogger(__name__)
 
 
 # Check if installed as editable
 def is_editable_install():
-    site_packages = site.getsitepackages()
-    return not any((pathlib.Path(sp) / "faebryk").exists() for sp in site_packages)
+    distro = Distribution.from_name(__name__.split(".")[0])
```

Comment:
Hit an issue and apparently this is a more robust way to detect editable installation: https://stackoverflow.com/questions/43348746/how-to-detect-if-module-is-installed-in-editable-mode

Reply:
Very nice!